### PR TITLE
Add fetch_size for pagination support

### PR DESCRIPTION
**Pull request by rupal-bq**
*Issue #77*

*Description of changes:*
- Added fetch_size in DSN configuration UI.
- Added fetch_size in the connection string.
- Added integration test for default and custom fetch size.

The default value for fetch_size is currently zero which is used for no cursor behaviour.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+        // Get column count
+        SQLSMALLINT total_columns = -1;
+        SQLNumResultCols(m_hstmt, &total_columns);
+        std::vector< std::vector< Col > > cols(total_columns);
+        for (size_t i = 0; i < cols.size(); i++)
+            cols[i].resize(SINGLE_ROW);
+
+        // Bind and fetch
+        for (size_t i = 0; i < cols.size(); i++)
```
Comment (jordanw-bq) [inline src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp:59]: Add curly braces, even for 1-line loops

File: src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    int GetTotalRowsAfterQueryExecution() {
+        SQLAllocHandle(SQL_HANDLE_STMT, m_conn, &m_hstmt);
+        SQLRETURN ret = SQLExecDirect(m_hstmt, (SQLTCHAR*)m_query.c_str(), SQL_NTS);
+        EXPECT_EQ(SQL_SUCCESS, ret);
+
+        // Get column count
+        SQLSMALLINT total_columns = -1;
+        SQLNumResultCols(m_hstmt, &total_columns);
+        std::vector< std::vector< Col > > cols(total_columns);
+        for (size_t i = 0; i < cols.size(); i++)
```
Comment (jordanw-bq) [inline src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp:55]: Add curly braces, even for 1-line loops

File: src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    std::string output = testing::internal::GetCapturedStdout();
+    std::cout << output << std::endl;
+    std::cout << (failures ? "Not all tests passed." : "All tests passed")
+              << std::endl;
+    WriteFileIfSpecified(argv, argv + argc, "-fout", output);
+
+#ifdef __APPLE__
+    // Disable malloc logging and report memory leaks
+    system("unset MallocStackLogging");
+    system("leaks itodbc_info > leaks_itodbc_info");
```
Comment (jordanw-bq) [inline src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp:146]: Update filename here
```suggestion
    system("leaks <executable-name> > leaks_<executable-name>");
```

File: src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp (hunk 1 above)
Comment (rupal-bq) [reply src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp]: added

File: src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp (hunk 2 above)
Comment (rupal-bq) [reply src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp]: added

File: src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp (hunk 3 above)
Comment (rupal-bq) [reply src/IntegrationTests/ITODBCPagination/test_odbc_pagination.cpp]: done

**Pull request merged**